NotebookAnalyzers/entities.py

```diff
@@ -23,9 +24,29 @@ def __init__(self, notebook_path: str):
         # Importing the notebook as a Python script
         transform_notebook(ipynb_file=notebook_path, export_list=["py"])
         notebook_path = notebook_path.split("/")[-1]
+        self.path = notebook_path
         with open(notebook_path.replace(".ipynb", ".py")) as f:
             self.script = f.read()
 
+    def get_info(self, bottom_size=4):
+        notebook_tuple = (self.path,
```

Comment:
@louieQ i write here the same I told you on Teams just to keep the pull updated. the function now returns a dict object containing all the the pynblint results. Only thing is that I could not mantain the structure of the API dict response with the nested dicts (statistics,linting) because I could not represent them in the dataframe(can't find a way to represent a list of nested dictionaries).